.github/workflows/cd-swift-cua-driver.yml

```
@@ -211,114 +183,123 @@ jobs:
+          find ./release-assets -name "*.tar.gz" -exec shasum -a 256 {} \; | sed 's|./release-assets[^/]*/||' >> combined_checksums.txt
+          echo '```' >> combined_checksums.txt
+          cat combined_checksums.txt
+
+          checksums=$(cat combined_checksums.txt)
+          echo "checksums<<EOF" >> $GITHUB_OUTPUT
+          echo "$checksums" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+
       - name: Create Release
-        if: startsWith(github.ref, 'refs/tags/cua-driver-v')
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)